tg/controllers.py

```diff
@@ -604,12 +610,16 @@ def _refresh_current_chat(self, current_chat_id: Optional[int]):
 
 
 def insert_replied_msg(msg: MsgProxy) -> str:
-    text = msg.text_content if msg.is_text else msg.content_type
-    return (
-        "\n".join([f"{REPLY_MSG_PREFIX} {line}" for line in text.split("\n")])
-        # adding line with whitespace so text editor could start editing from last line
-        + "\n "
-    )
+    if text := msg.text_content if msg.is_text else msg.content_type:
+        return (
+            "\n".join(
+                [f"{REPLY_MSG_PREFIX} {line}" for line in text.split("\n")]
+            )
+            # adding line with whitespace so text editor could start editing from last line
+            + "\n "
+        )
+    else:
```

Comment:
There is no need to else.
I am in the part of programmers which trying to avoid using else as much as possible and using early returns.
I think you understand benefits from early returns and there is no need to explain them.

Reply:
>  I think you understand benefits from early returns and avoiding "else" and there is no need to explain them.

no, i can't see any befits for this particular case, so you can explain it a bit more 🙂 .

however, both approaches looks similar to me, so I removed else here

Reply:
usually people avoid `else` to reduce nesting, so it could bee the only benefit here. readability is the same for both approaches.